### DISCUSSION TITLE
Training YoloV8 on pascal VOC2007 seems not to converge to anything usefull

### DISCUSSION THREAD

**Discussion by kasper64** [keras-team/keras-cv#2439] [Q&A]
Hello,
I'm following a tutorial from the Keras-CV documentation: https://keras.io/guides/keras_cv/object_detection_keras_cv/
I'm following the advanced section. The first few epochs, the coco metrics show all around 0, but after a few epochs more, they start to to show `-1`. It seems to stay like this forever. I tested as far as epoch 200. When I use the model, it always outputs 0 detected objects. The loss function settles around 3 very quickly, and doesn't change as well.

Maybe I am not patient enough, but the tutorial mentions that 10 epochs can already give good results. 

When I load a pretrained model ("yolo_v8_m_pascalvoc") it does work on my setup.

I attached the code I'm using. It's basically the code from the tutorial, but condensed and the beginner part removed.

I tested it on two different linux systems with different nvidia driver versions. (550 and 450)

The code I use:  (including a `requirements.txt`) [keras_yolov8_tutorial.zip](https://github.com/keras-team/keras-cv/files/15198662/keras_yolov8_tutorial.zip)

Package versions I use:
> keras==3.2.0
> keras-cv==0.9.0
> pycocotools==2.0.7
> scikit-image==0.23.2
> tensorflow==2.16.1

The reason I am trying to get the tutorial working is because I get the same results on my own dataset.
